Gems/GraphCanvas/Code/Source/Widgets/NodePropertyDisplayWidget.cpp

```diff
@@ -201,6 +189,9 @@ namespace GraphCanvas
         // remove an item from the scene while that is in progress, it can crash since it is
         // still internally using a cached list of the scene items.
         QTimer::singleShot(0, this, [this, isForcedEdit]() {
+            // There is only one m_layoutItem in m_layout, so we are ok to clear layout in this timer event
```

Comment:
As you've pointed out there's only ever one item in the layout, which begs to question why we even need a layout...but that's a problem for another PR ;) Excellent work!